src/main/kotlin/com/deviceinsight/helm/TemplateMojo.kt

```diff
@@ -64,6 +64,10 @@ class TemplateMojo : AbstractHelmMojo() {
 				file = project.basedir.resolve(file)
 			}
 
+			if (!file.parentFile.exists()) {
+				file.parentFile.mkdir()
```

Comment:
Mind using `mkdirs()` instead? https://docs.oracle.com/javase/8/docs/api/java/io/File.html#mkdirs()

Reply:
actually this was done intentionally. the directory above is always the target directory and it should always be there

Reply:
But is there any drawback in using `mkdirs()`?

> the directory above is always the target directory and it should always be there

How about running `mvn helm:template` from a clean repository? Does Maven always create the target directory first? I'd argue that using `mkdirs()` doesn't hurt here.